ui/raidboss/data/07-dt/raid/r7s.ts

```diff
@@ -509,6 +509,149 @@ const triggerSet: TriggerSet<Data> = {
         'Winding Wildwinds/Crossing Crosswinds': 'Wildwinds/Crosswinds',
       },
     },
+    {
+      'locale': 'de',
+      'missingTranslations': true,
+      'replaceSync': {
+        'Blooming Abomination': 'Biestinator-Spross',
+        'Brute Abombinator': 'Brutalo Biestinator',
+      },
+      'replaceText': {
+        '\\(adds': '(Adds',
+        'cast\\)': '(Wirken)',
```

Comment:
It looks like the parenthesis pairs don't match.

Reply:
true, i needed to change things around to avoid some edge cases and forgot to remove it here